wrangler-service/src/main/java/co/cask/wrangler/service/adls/ADLSHandler.java

```diff
@@ -196,9 +198,7 @@ public void loadADLSFile(HttpServiceRequest request, HttpServiceResponder respon
                 return;
             }
 
-            if (Strings.isNullOrEmpty(scope)) {
-                scope = WorkspaceDataset.DEFAULT_SCOPE;
-            }
+            String group = WorkspaceUtils.getScope(scope, request.getHeader(USER_ID_KEY), getContext().getRuntimeArguments());
```

Comment:
Please rename the local variable 'group' to something more appropriate eg. 'effectiveScope' as the method which is returning it is 'getScope'

Reply:
done, thanks.